### PR TITLE
Modified permission to User.Read.All

### PR DESCRIPTION
**Pull request by piotrci**
Does the app truly require the ability to write to all users in the directory? if not, than User.Read.All should be sufficient to receive notifications. Yes, it still requires admin consent.

### CODE REVIEW

**Comment by piotrci** [comment]
@jasonjoh , can you please have a look? is User.Read**Write**.All truly required for the demo app?

File: tutorial/02_create-app.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ In this exercise, you will create a new Azure AD web application registration us
 
     ![Screenshot selecting the Microsoft Graph service](./images/aad-portal-newapp-graphscope.png)
 
-    Select **Application Permission**, expand the **User** group and select **User.ReadWrite.All** scope.
+    Select **Application Permission**, expand the **User** group and select **User.Read.All** scope.
 
     Select **Add permissions** to save your changes.
 
     ![Screenshot selecting Microsoft Graph scope](./images/aad-portal-newapp-graphscope-02.png)
```
Comment (piotrci) [inline tutorial/02_create-app.md]: @jasonjoh , if you agree with the change i proposed, can you also update the screenshots to reflect it correctly?

**Comment by piotrci** [comment]
#hold-off

**Comment by jasonjoh** [comment]
@andrewconnell do you know if ReadWrite is required?

**Comment by andrewconnell** [comment]
Offhand I don't know. I seem to recall some issues without this permission where changing some properties (office location / phone) weren't triggering webhook notifications, but it's been a while and I don't recall.

**Comment by jasonjoh** [comment]
Closing this as this was changed as part of quarterly refresh in #15

**Pull request closed without merging**